examples/functional-iteration/tests/functional_iteration_test.bal

```diff
@@ -0,0 +1,43 @@
+import ballerina/test;
+import ballerina/io;
+
+any[] outputs = [];
+int counter = 0;
+
+// This is the mock function which will replace the real function
+@test:Mock {
+    moduleName: "ballerina/io",
+    functionName: "println"
+}
+public function mockPrint(any... s) {
+    foreach any a in s {
+        outputs[counter] = a;
+        counter += 1;
+    }
+}
+
+@test:Config
+function testFunc() {
+    // Invoking the main function
+    main();
+    test:assertEquals(outputs.length(), 18);
+    test:assertEquals(outputs[0], "Number of elements in 'words': ");
+    test:assertEquals(outputs[1], 5);
+    string[] s = ["ANT", "BEAR", "CAT", "DEAR", "ELEPHANT"];
```

Comment:
Instead of s. Would it make sense to use a more meaningful name i.e animals ?

Reply:
Updated to use inline with c549784, thanks!